internal/manager/serverless/native.go

```diff
@@ -0,0 +1,152 @@
+package serverless
+
+import (
+	"fmt"
+	"io"
+	"os/exec"
+	"path/filepath"
+)
+
+func (sls *Manager) deployWithNative(w io.Writer) error {
+	err := sls.runNpmInstall(w)
+	if err != nil {
+		return fmt.Errorf("can't run nvm: %w", err)
+	}
+
+	err = sls.runNpmInstall(w)
+	if err != nil {
+		return fmt.Errorf("can't run npm install: %w", err)
+	}
+
+	if sls.App.CreateDomain {
+		err = sls.runCreateDomain(w)
+		if err != nil {
+			return fmt.Errorf("can't run serverless create_domain: %w", err)
+		}
+	}
+
+	err = sls.runDeploy(w)
+	if err != nil {
+		return fmt.Errorf("can't run serverless deploy: %w", err)
+	}
+
+	return nil
+}
+
+func (sls *Manager) runNpmInstall(w io.Writer) error {
+	//npmInstallCommand := "npm install --save-dev"
+
+	cmd := exec.Command("npm", "install", "--save-dev")
+	cmd.Stdout = w
+	cmd.Stderr = w
+	cmd.Dir = filepath.Join(sls.App.Path)
+	err := cmd.Run()
+	if err != nil {
+		return fmt.Errorf("can't run nvm: %w", err)
+	}
+
+	return nil
+}
+
+func (sls *Manager) runNvm(w io.Writer) error {
+	nvmCommand := fmt.Sprintf("source $HOME/.nvm/nvm.sh && nvm install %s && nvm use %s", sls.App.NodeVersion, sls.App.NodeVersion)
```

Comment:
Let's try to rely on `NVM_DIR` if it exists in the current shell. If not let's use `$HOME` from the sub-shell you're creating.

I haven't tested this code but it should probably work:
```suggestion
        nvmDir := os.Getenv("NVM_DIR", "$HOME/.nvm")
	nvmCommand := fmt.Sprintf("source %s/nvm.sh && nvm install %s && nvm use %s", nvmDir, sls.App.NodeVersion, sls.App.NodeVersion)
```